CSS/style.css

```diff
@@ -402,8 +515,54 @@ hr {
   left: -20px;
 }
 
+@media only screen and (min-width: 995px) {
```

Comment:
- [ ] You have 3 `@media` with the same range aspect, but you should have different ratios for `tablets`, and that been said the view on tablets more is a bit out of range a lot of margin on the right side.

![Capture web_30-9-2021_171451_raw githack com](https://user-images.githubusercontent.com/61562626/135493113-c660cb56-745f-469d-a171-0628aec15476.jpeg)